packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItemWrapper/ContextualMenuSplitButton.tsx

```diff
@@ -56,10 +59,11 @@ export class ContextualMenuSplitButton extends ContextualMenuItemWrapper {
           <div
             data-ktp-target={ keytipAttributes['data-ktp-target'] }
             ref={ (splitButton: HTMLDivElement) => this._splitButton = splitButton }
-            role={ 'button' }
-            aria-labelledby={ item.ariaLabel }
+            role={ 'menuitem' }
+            aria-label={ item.ariaLabel ? item.ariaLabel : item.name }
```

Comment:
do we want to default to name? Or would we rather just use ariaLabel if it's there, who knows what someone may but on the name since it's not supposed to be used for anything accessibility related

Reply:
I got rid of it. I swore I saw somewhere in the definition for aria-label that says that we should use name be default if aria-label is empty. Looking back at it I misunderstood it.